composer.lock

```diff
@@ -3447,16 +3518,16 @@
         },
         {
             "name": "vimeo/psalm",
-            "version": "4.27.0",
+            "version": "4.29.0",
```

Comment:
I think the problem could be the update to this dependency. Perhaps if you install pint without updating psalm, there won't be any errors.

Reply:
Sure. Will make this change and let you know

Reply:
Done